src/models/game_piece_action.ts

```diff
@@ -23,7 +25,7 @@ export type GamePieceRangedAttackAction = {
   actionType: 'rangedAttack';
   resolved: boolean;
   targetGamePieceId: number;
-  encodedDiceRolls: EncryptedDiceRolls;
+  encryptedAttackRolls: EncrytpedAttackRoll[];
```

Comment:
Actually, I'm not sure if I can do this.  This type is "snarky", so might be better to stick with the version based on strings and "snarkify" it later.